linters/htmlhint.js

```diff
@@ -0,0 +1,190 @@
+'use strict';
+
+const htmlhint = require('htmlhint').HTMLHint;
+const { flatten, parseJson, readFiles, toArray } = require('../helper');
+
+const config = parseJson(__dirname + '/../config/htmlhint.hjson');
+
+// Add our custom rules.
+htmlhint.addRule({
+	id: 'doc-lang',
+	description: '<html> tags must have a "lang" attribute.',
+	init: configureDocLang
+});
+htmlhint.addRule({
+	id: 'link-href',
```

Comment:
That's an idea. Speaking of documentation, I should probably add docs about these rules to the project.